### ISSUE TITLE
[BUG] Accessibility - Web App - Automatic video plays and flashes upon refresh

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4864]
**Describe the bug**
On https://api.rokwire.illinois.edu/illinois/
Is this video supposed to automatically play when users refresh their screens?
If so, it needs a mechanism to play/pause like bug #4863 
WCAG Success Criterion 2.2.2, known as Pause, Stop, Hide

Additionally nothing is announced to the screen reader users. 
For videos, WCAG recommends the following:
Captions: Provide synchronized captions for all audio content in videos to make them accessible to people who are deaf or hard of hearing.
Audio Descriptions: Provide audio descriptions of important visual information in videos for people who are blind or have low vision.
Transcripts: Provide transcripts for both audio and video content. Transcripts should include all spoken dialogue and important visual information.
These requirements are part of WCAG's Guideline 1.2: Time-based Media

**To Reproduce**
1. Go to 'https://api.rokwire.illinois.edu/illinois/' in Chrome Browser ( Windows 11)
2. Video begins playing 

**Expected behavior**
Any automatically playing video needs a play/pause mechanism

**Actual behavior**
Currently the video autoplays and there are no Captions, Audio Description, or transcript. 

**Screenshots**

https://github.com/user-attachments/assets/ddb62a5d-c88b-4ce6-be9c-a4aa30a814fc

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Firefox
 - Version Version 136.0 (64-bit) 

**Additional context**

**Comment by acovher2** [comment]
For this, the video is basically a loading indicator. For the sake of those with seizure disorders or vestibular conditions that make them ill with motion on screen, we need a pause/play mechanism to stop the motion. This needs to be receiving focus upon loading video beginning. Additionally, screen reader users need something announced to them indicating "page loading"

**Comment by acovher2** [comment]
Verified as still existing in 7.0.5 version on: UIUC 7.0 Dev Web
https://api-dev.rokwire.illinois.edu/illinois/

**Comment by dobromirdobrev** [comment]
@acovher2 , this should be handled in version 7.3.22 of the Illinois web app available on dev environment here: https://api-dev.rokwire.illinois.edu/illinois/ .

So, we now have pause/play button that receives focus when the video starts. You can verify it when you have a chance.

**Comment by acovher2** [comment]
@dobromirdobrev Ah, forgive me, this has the button therefore will close as 2.1.1 WCAG is handled in 5136 as you mentioned. 
Thank you!
